src/components/utils/unit_tests/onfidoApi.test.js

```diff
@@ -0,0 +1,81 @@
+import { sendMultiframeSelfie } from "../onfidoApi"
+
+let jwtToken = 'my_token'
```

Comment:
You don't need to override this, so `const` would be more appropriate.

Reply:
True. I was overriding in a test that I canceled and I forgot to change it back. Thanks